DTS-STN/Alpha-Site
# Update alt text

# Description

[63994 - DEV: Verify alt text for all images follows best practices](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=63994)

Pretty meaty PR that takes care of a couple things. Firstly, I've updated alt text across the site where necessary so it follows best practices. Most of this was done directly in AEM however I have removed alt text (ie `alt=""`) on images that are purely decorative manually. Since the `/notsupported` page has browser icons which needed updated alt text, I overhauled the error page model in AEM to leverage the SCLabs-Image model which has fields for alt text and so the field names followed our agreed upon naming convention (`scSomethingSomething` vs `sclSomethingSomething`). Doing this required that I update all the error pages so they are up to date with the new model. 

Lastly, I've updated the `/thankyou` page to use the animated checkmark Lottie file instead of the static checkmark SVG since it made no sense that we had two separate images for checkmarks. I've also ensured that the animated checkmark has descriptive text that is announced to the screen reader. 


## Test Instructions

1. Pull in branch
2. Type `yarn dev`
3. Navigate through all the pages and ensure images that aren't purely decorative have alt text that follows best practices (based on the link provided in the task description)

## Checklist


## Product and Sprint Backlog

[Azure](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities)
